Products/LDAPMultiPlugins/ActiveDirectoryMultiPlugin.py

```diff
@@ -205,7 +205,13 @@ def getGroupsForPrincipal(self, user, request=None, attr=None):
         filt = '(&(objectClass=%s)(|%s))' % (self.group_class, ''.join(cn_flts))
 
         delegate = acl._delegate
-        R = delegate.search(acl.groups_base, acl.groups_scope, filter=filt)
```

Comment:
Doesn't the same code in `_recurseGroups()` also need to consider `lookup_groups_base`?